addons/survey/controllers/main.py

```diff
@@ -302,8 +288,8 @@ def survey_reporting(self, survey, token=None, **post):
             filter_finish = True
         if post or filter_finish:
             filter_data = self.get_filter_data(post)
-            current_filters = survey_obj.filter_input_ids(request.cr, request.uid, survey, filter_data, filter_finish, context=request.context)
-            filter_display_data = survey_obj.get_filter_display_data(request.cr, request.uid, filter_data, context=request.context)
+            current_filters = Survey.filter_input_ids(survey, filter_data, filter_finish)
```

Comment:
Same as above, why not call it with survey and change signature of filter_input_ids ?